play-akka-http-ws-standalone/src/main/java/play/libs/ws/akkahttp/StandaloneAkkaHttpWSRequest.java

```diff
@@ -0,0 +1,585 @@
+/*
+ * Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package play.libs.ws.akkahttp;
+
+import akka.actor.ActorSystem;
+import akka.http.impl.model.parser.HeaderParser$;
+import akka.http.javadsl.Http;
+import akka.http.javadsl.model.*;
+import akka.http.javadsl.model.headers.BasicHttpCredentials;
+import akka.http.scaladsl.model.HttpHeader;
+import akka.http.scaladsl.model.HttpHeader$;
+import akka.pattern.PatternsCS;
+import akka.stream.Materializer;
+import play.libs.ws.*;
+import scala.concurrent.duration.FiniteDuration;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public final class StandaloneAkkaHttpWSRequest implements StandaloneWSRequest {
+
+  private final HttpRequest request;
+  private final List<WSRequestFilter> filters;
+  private final Duration timeout;
+
+  private final ActorSystem sys;
+  private final Materializer mat;
+
+  StandaloneAkkaHttpWSRequest(String url, ActorSystem sys, Materializer mat) {
+    this.request = HttpRequest.create(url);
+    this.filters = new ArrayList<>();
+    this.timeout = Duration.ZERO;
+    this.sys = sys;
+    this.mat = mat;
+  }
+
+  private StandaloneAkkaHttpWSRequest(HttpRequest request, List<WSRequestFilter> filters, Duration timeout, ActorSystem sys, Materializer mat) {
+    this.request = request;
+    this.filters = filters;
+    this.timeout = timeout;
+    this.sys = sys;
+    this.mat = mat;
+  }
+
+  /**
+   * Perform a GET on the request asynchronously.
+   *
+   * @return a promise to the response
+   */
+  @Override
+  public CompletionStage<? extends StandaloneWSResponse> get() {
+    return execute("GET");
```

Comment:
Could we use the constant values here where we know it statically like `HttpMethods.GET`?

Reply:
Done in https://github.com/playframework/play-ws/pull/205/commits/f00d8b94256ff4246764b3ac78b0de44dfbbbbb6